webclient/src/agave.ts

```diff
@@ -551,7 +556,7 @@ export class AgaveClient {
    * @param g The green value between 0 and 1
    * @param b The blue value between 0 and 1
    */
-  background_color(r: number, g: number, b: number) {
+  backgroundColor(r: number, g: number, b: number) {
```

Comment:
Any reason for these setter methods being `backgroundColor` vs. `setBackgroundColor`?

Reply:
I'm not going to change this right now because I'd have to change it in many many places.  The basic reasoning is that early on, almost every command was a "set" command so why bother saying set all the time.  But gradually I forgot and added some with the word set so it's an inconsistent mix now.   Good idea to clean it up in a future PR.  Probably will make some "set" command names that just alias to the old ones so as not to break anything.